src/util/settings.cpp

```diff
@@ -60,9 +60,16 @@ bool ReadSettings(const fs::path& path, std::map<std::string, SettingsValue>& va
     values.clear();
     errors.clear();
 
+    if (!fs::exists(path)) {
+        return true; // Ok for file to not exist
+    }
```

Comment:
This should be a single line as per https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c (bullet point #7)

Also, the comment should be above the line(even though there are lines that stray away from this):
```
// Ok for file to not exist
if (!fs::exists(path)) return true;
```

Reply:
Bullet point 7 says it *can* be on a single line, but it doesn't have to

Reply:
I think that aids readability. Fixed.